index.bs

```diff
@@ -3181,6 +3251,10 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. Set |source|'s [=attribution source/number of event-level reports=] to 0 if
     |source|'s [=attribution source/randomized response=] is null,
     [=attribution source/randomized response=]'s [=list/size=] otherwise.
+1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] obtained with |randomizedResponseConfig|.
+1. If |sourceType| is "<code>[=source type/event=]</code>" and |states| is greater than |source|'s [=attribution source/max event states=]:
+    1. Run [=obtain and deliver debug reports on source registration=] with "<code>[=source debug data type/source-max-event-states-limit=]</code>" and |source|.
+    1. Return.
```

Comment:
Considering that the number of states is calculated in `computing the channel capacity of a source` above as well, would it make more sense to move this to be before calculating channel capacity and pass `|states|` into the computation? That way we prioritize `source-max-event-states-limit` over the channel capacity limit like we do for `source-trigger-state-cardinality-limit`

Reply:
this should be addressed in the information gain PR: https://github.com/WICG/attribution-reporting-api/pull/1364/files